g11n-ws/modules/md-service-i18n-l3/src/main/java/com/vmware/vip/core/messages/service/product/ProductService.java

```diff
@@ -166,8 +167,31 @@ private DropVersionDTO getDropVersion(DropVersionDTO versioninfo,JSONObject jo,
 						+ componentMessagesDTO.getVersion(), e);
 			}
 		}
+		updateMultComponentCache(componentMessagesDTOList.get(0).getProductName(), componentMessagesDTOList.get(0).getVersion());
 		return translationDTOList;
 	}
+	
+	/**
+	 * update the MultComponent Cache project when update translations
+	 * 
+	 * @param productName
+	 * @param version
+	 * @throws L3APIException
+	 */
+	private void updateMultComponentCache(String productName, String version) throws L3APIException {
+		com.vmware.vip.core.messages.service.multcomponent.TranslationDTO multComp = new com.vmware.vip.core.messages.service.multcomponent.TranslationDTO();
+		multComp.setProductName(productName);
+		multComp.setVersion(version);
+		multComp.setLocales(getSupportedLocaleList(productName, version));
+		multComp.setComponents(getComponentNameList(productName, version));
+		String multKey = CachedKeyGetter.getMultiComponentsCachedKey(multComp);
+		try {
+			TranslationCache3.deleteCachedObject(CacheName.MULTCOMPONENT, multKey,
+					com.vmware.vip.core.messages.service.multcomponent.TranslationDTO.class);
+		} catch (VIPCacheException e) {
+			throw new L3APIException("MULTCOMPONENT cache occurs error when update translation.", e);
```

Comment:
Here should log an error message rather than throwing an exception because translation has been updated successfully.

Reply:
```suggestion
			logger.Error("Error occurs when deleting MULTCOMPONENT cache.");
```

Reply:
Here we should throw an exception. Because we have an ExceptionHandle class to Unify handling the level3 error

Reply:
We should keep this error internally because translation has been updated successfully. If users get this error, they will be confused.